scripts/zones/Cape_Teriggan/Zone.lua

```diff
@@ -14,7 +14,7 @@ require("scripts/globals/zone")
 function onInitialize(zone)
     UpdateNMSpawnPoint(ID.mob.KREUTZET)
     GetMobByID(ID.mob.KREUTZET):setRespawnTime(math.random(32400,43200)) -- 9 to 12 hours
```

Comment:
This is a good time to go ahead and make qa local variable for this so that the core doesn't have to do the lookup multiple times here.

```lua
local mob = GetMobByID(ID.mob.KREUTZET)
```

Reply:
I did it in both spots... that's good, i need to start learning to pay attention to performance now.